package application;

import java.util.List;

import model.dao.DaoFactory;
import model.dao.SellerDao;
import model.entities.Department;
import model.entities.Seller;

public class Program {

	public static void main(String[] args) {
		Department obj = new Department(1, "Books");
		
		//Seller seller = new Seller(21, "Bob", "Bob@gmail.com", new Date(), 3000.00, obj);
		
		SellerDao sellerDao = DaoFactory.createSellerDao();
		
		System.out.println("=== TEST 1: selller findById =====");
		Seller seller = sellerDao.findById(3);
		System.out.println(seller);
		System.out.println("=== TEST 2: selller findByDepartment =====");
		Department department = new Department(2, null);
		List<Seller> list = sellerDao.findByDepartment(department);
		list.forEach(System.out::println);
		
		System.out.println("=== TEST 2: selller findByAll =====");
		List<Seller> list2 = sellerDao.findAll();
		list2.forEach(System.out::println);

	}

}
 